tensorboard/backend/event_processing/plugin_event_accumulator.py

```diff
@@ -252,6 +258,21 @@ def FirstEventTimestamp(self):
             except StopIteration:
                 raise ValueError("No event timestamp could be found")
 
+    @property
+    def SourceWriter(self) -> Optional[str]:
```

Comment:
style nit: although CamelCase naming is usually allowed on methods for consistency within a file, this is an @property, which is generally accessed like a class variable, so having a callsite like `accumulator.SourceWriter` is IMO pretty confusing to read (it looks like a nested class or something, not a property). I would tend to stick with lower_with_underscores style for any @property, and suppress the lint warning if need be.

Alternatively, you could change to GetSourceWriter() like on the Multiplexer. (It would seem nicer to have them use the same API for consistency, regardless.)